src/main/scala/cognite/spark/v1/SyncCursorCallback.scala

```diff
@@ -0,0 +1,60 @@
+package cognite.spark.v1
+
+import cats.effect.IO
+import com.cognite.sdk.scala.sttp.{GzipBackend, RetryingBackend}
+import io.circe.Codec
+import io.circe.generic.semiauto.deriveCodec
+import scala.concurrent.duration._
+import sttp.client3.{SttpBackend, basicRequest}
+import sttp.client3.asynchttpclient.SttpClientBackendFactory
+import sttp.client3.asynchttpclient.cats.AsyncHttpClientCatsBackend
+import sttp.client3.circe.{asJson, circeBodySerializer}
+import sttp.model.{MediaType, Uri}
+
+case class IncrementalCursor(name: String, value: String)
+case class IncrementalCursorResponse(name: String, value: String, updated: Boolean)
+
+object SyncCursorCallback {
+  @transient private lazy val sttpBackend: SttpBackend[IO, Any] =
+    new GzipBackend[IO, Any](
+      AsyncHttpClientCatsBackend.usingClient(SttpClientBackendFactory.create("Last-Cursor-Submitter")))
+
+  @transient private val retryingBackend = new RetryingBackend[IO, Any](
```

Comment:
```suggestion
  @transient private lazy val retryingBackend = new RetryingBackend[IO, Any](
```
otherwise it'd force the other lazy field anyway

Reply:
Indeed. Merged.